README.md

```diff
@@ -52,6 +52,7 @@ We welcome any vendor to fork the project to demonstrate their services and
 adding a link below. The community is committed to maintaining the project and
 keeping it up to date for you.
 
+- [Appdynamics](https://www.appdynamics.com/blog/cloud/how-to-observe-opentelemetry-demo-app-in-appdynamics-cloud/)
```

Comment:
Should the `D` by uppercase?

![image](https://user-images.githubusercontent.com/17327289/202773864-2a836a4e-8108-4f94-8894-a4db9e817b5f.png)

Reply:
![image](https://user-images.githubusercontent.com/17327289/202775880-e1faefd2-a55e-47f1-b82b-b77915c3ea83.png)

Reply:
Thankyou @reyang for catching this ! Will create a fix soon.

Reply:
Created PR for the fix: https://github.com/open-telemetry/opentelemetry-demo/pull/595